### ISSUE TITLE
ADD LC RWO Lookups

### ISSUE THREAD

**Issue by sfolsom** [cul-it/qa_server#395]
For vocabs that model RWOs in id.loc.gov (and we're using those URIs in application profiles) add lookups.

Waiting on LOC to provide a list, but we know LCNAF (minus Name/Titles and Titles) will need this.

**Comment by sfolsom** [comment]
There's an API for RWOs: https://id.loc.gov/rwo/agents/suggest2/?q=. This API doesn't currently allow filtering by type (Persons, Orgs, etc.) 

This is different than what the non-RWO lookups use in the Samvera Gem, see: https://github.com/cul-it/qa_server/issues/364. Perhaps we could create a new lookup that builds off the gem, swapping out URIs starting with http://id.loc.gov/authorities/names/ with http://id.loc.gov/rwo/agents/

If we go with the latter we'll need:
- [ ] Persons
- [ ] Conferences
- [ ] Organizations
- [ ] Geographics
- [ ] Families

**Comment by sfolsom** [comment]
I'm just remembering now that for the non-RWO lookups we already had to override the Samvera results because they started with "info:...", see: https://github.com/samvera/questioning_authority/wiki/Connecting-to-Library-of-Congress-%28LOC%29. This new lookup just add http://id.loc.gov/rwo/agents/ prefix instead of http://id.loc.gov/authorities/names/.

**Comment by Baroquem** [comment]
For my own reference:
https://id.loc.gov/search/?q=memberOf:http://id.loc.gov/authorities/names/collection_LCNAF

Persons -> rdftype:PersonalName
Conferences -> ConferenceName
Organizations -> CorporateName
Geographics -> Geographic
Families -> FamilyName

**Comment by Baroquem** [comment]
@sfolsom I just realized that this seems to be already implemented -- see

http://localhost:3000/authorities/search/loc/names_Geographic?q=Cambridge
http://localhost:3000/authorities/search/loc/names_ConferenceName?q=IEEE
http://localhost:3000/authorities/search/loc/names_CorporateName?q=apple
http://localhost:3000/authorities/search/loc/names_FamilyName?q=baker
http://localhost:3000/authorities/search/loc/names_PersonalName?q=Connolly,Matthew

This isn't using the rwo/agents prefix, though.

**Comment by sfolsom** [comment]
Right. Sorry I should have mentioned that. We just need the results returned with the rwo URL.

**Comment by Baroquem** [comment]
@sfolsom Sorry for the delay! Please approve https://github.com/LD4P/qa_server_container/pull/96, and we'll be able to merge and test.

**Comment by Baroquem** [comment]
This is available now on int, e.g. http://lookup-int.ld4l.org/authorities/search/loc/agents_Geographic?q=Cambridge
